backend/papersfeed/tests/tests_user.py

```diff
@@ -107,7 +107,7 @@ def test_sign_in(self):
 
         self.assertIn(
             '{"id": ' + str(user_id)
-            + ', "username": "swpp", "email": "swpp@snu.ac.kr", "description": "",'
+            + ', "username": "swpp", "email": "swpp@snu.ac.kr", "description": "", "photo_index": 0,'
```

Comment:
You can use assertJSONEqual for simple codes!

Reply:
I just added a word on the previous code, but thanks for letting me know.